pkg/core/interop/context.go

```diff
@@ -62,10 +62,12 @@ func NewContext(trigger trigger.Type, bc blockchainer.Blockchainer, d dao.DAO, n
 // it's supposed to be inited once for all interopContexts, so it doesn't use
 // vm.InteropFuncPrice directly.
 type Function struct {
-	ID    uint32
-	Name  string
-	Func  func(*Context, *vm.VM) error
-	Price int64
+	ID   uint32
+	Name string
+	Func func(*Context, *vm.VM) error
+	// ParamCount is a number of function parameters.
+	ParamCount int
```

Comment:
`System.Runtime.GetNotifications` has 1 parameter. (Just cannot put the comment in the right place)